index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>
+    Document
+  </title>
+</head>
+
+<body>
+  <div class=border>
+    <h1> Jaye Lin Resume </h1>
+    <div class="contact">
+      <p> <a href="Illusxia. carbonmade.com"> Portfolio </a><br>
+
+        <a href="Xiaoxxia23@Gmail.com"> Email </a> <br>
+        Baltimore, MD
+      </p>
+    </div>
+
+    <div class="edu">
+      <h3> Education And Training: </h3>
+      <p>
+        <b>Maryland Institute Of Art And Design</b>
+        <br />
+        <i>(2019-Current)</i><br />
```

Comment:
You should review the MDN usage notes for the `b` and `i` elements and judge if they import the right meaning you intend for this text:
- [`b`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b)
- [`i`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i)

Additionally, the [`br`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) element is inappropriate here and you should consider updating your markup to work without it.